crates/router/src/core/cache.rs

```diff
@@ -0,0 +1,16 @@
+use common_utils::errors::CustomResult;
+
+use super::errors;
+use crate::{cache::CONFIG_CACHE, db::StorageInterface, services};
+
+pub async fn invalidate(
+    store: &dyn StorageInterface,
+    key: &str,
+) -> CustomResult<services::api::ApplicationResponse<serde_json::Value>, errors::ApiErrorResponse> {
+    CONFIG_CACHE.remove(key).await;
```

Comment:
@jeevaprakashdr can you add `ACCOUNT_CACHE.remove(key).await`

Reply:
ahh okie, now it is clear a new in-memory cache has been introduced as part of  [this change](https://github.com/juspay/hyperswitch/commit/da4d721424d329af618a63034aabe2d9248eb041) and it should be invalidated through the new endpoint. 

The changes are done and pushed.